km/km_hcalls.c

```diff
@@ -1971,7 +1971,7 @@ static km_hc_ret_t snapshot_putdata_hcall(void* vcpu, int hc, km_hc_args_t* arg)
 
 static km_hc_ret_t shrink_payload_hcall(void* vcpu, int hc, km_hc_args_t* arg)
 {
-   int km_shrink_footprint(km_vcpu_t* vcpu);
+   int km_shrink_footprint(km_vcpu_t * vcpu);
```

Comment:
The one place I get our coding style correct and it is changed to something different?

Reply:
This prototype can be removed since it is now in km.h.